### PR TITLE
[Contact Us]

### PR DESCRIPTION
**Pull request by ghost**
This is a html page about contacting us.

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<html>
+	<head><title>Contact Us</title></head>
```
Comment (ultimatecoder) [inline contact.html]: Please write each HTML tag in new line.

File: contact.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+	<body>
+		<h4>You can contact us at docs@kskv.com</h4>
```
Comment (ultimatecoder) [inline contact.html]: Warp each email under an anchor tag. For example `<a href="mailto:docs@kskv.com">docs@kskv.com</a>

**Review by ultimatecoder — COMMENTED** [review]
Good code, but please improve a bit

**Review by ghost — COMMENTED** [review]
HTML tags are written in different lines and anchor tag added for mail

**Pull request merged**